### DISCUSSION TITLE
How to set up in godot

### DISCUSSION THREAD

**Discussion by TheRealMG64** [Techokami/SonicWorldsNext#31] [Q&A]
how do i set it up in godot?

**Comment by DimensionWarped** [comment]
It's pretty straight forward honestly. You just download Godot 3.5, checkout the main branch of the project, and load it in Godot 3.5 like you would any other project. From there you can hit play and it'll go.

I wouldn't recommend trying to load it in any other versions of Godot right now though. @Renhoex is working on getting a Godot 4 branch ready currently, but it wouldn't do to make that your first experience with the engine as it is right now.
